ts/ts-diabetes/src/main/java/se/inera/intyg/common/ts_diabetes/v4/model/converter/certificate/question/QuestionBedomningUppfyllerBehorighetskrav.java

```diff
@@ -75,77 +75,77 @@ public static CertificateDataElement toCertificate(Bedomning bedomning, int inde
             .index(index)
             .config(
                 CertificateDataConfigCheckboxMultipleCode.builder()
-                    .text(texts.get(BEDOMNING_UPPFYLLER_BEHORIGHETSKRAV_TEXT_ID))
+                    .text(texts.get(BEDOMNING_UPPFYLLER_BEHORIGHETSKRAV_TEXT))
```

Comment:
Since this text is coming directly from RespConstants, we don't need to use the textprovider to get it. Instead we can just provide the plain text to the config.

Reply:
Mistake by me not to remove, thanks for spotting it